compiler_opt/distributed/local/local_worker_manager.py

```diff
@@ -208,16 +217,25 @@ def __dir__(self):
   return _Stub()
 
 
+@gin.configurable
 class LocalWorkerPool(AbstractContextManager):
   """A pool of workers hosted on the local machines, each in its own process."""
 
-  def __init__(self, worker_class: 'type[Worker]', count: Optional[int], *args,
+  def __init__(self,
+               worker_class: 'type[Worker]',
+               count: Optional[int],
+               *args,
+               pool_threads: int = 1,
                **kwargs):
     if not count:
       count = multiprocessing.cpu_count()
     self._stubs = [
-        _make_stub(worker_class, *args, **kwargs) for _ in range(count)
+        _make_stub(worker_class, pool_threads, *args, **kwargs)
+        for _ in range(count // pool_threads)
```

Comment:
nit: for _ in range(max(1, count//pool_threads))

... or logging.error if that nr is 0?

Reply:
a, nevermind - saw the code below